ui/themes/rjsf.js

```diff
@@ -30,15 +30,15 @@ export const rjsfTheme = createTheme({
         "&:hover" : "00b39f"
       }
     },
-    MuiInput : {
+    MuiTextField : {
       root : {
-        width : '70%',
+        marginRight : "0.5rem"
       }
     },
     MuiInputLabel : {
       root : {
         whiteSpace : "nowrap",
-        overflow : "hidden",
+        // overflow : "hidden",
```

Comment:
This Comment is on purpose would fix it.

Reply:
Does this need undone? Does the textOverflow : "ellipsis" below not take effect appropriately? It didn't seem to on today's call.

Reply:
Needs fixed, I think.